source/02-combat-prototype/actors/combat/CombatActions.gd

```diff
@@ -0,0 +1,38 @@
+extends Command
+
+var current setget set_current
+var _combat_action
```

Comment:
Maybe add an `onready` and set it to be one of the children of this class. Just so we have something to work with right at the beginning